cms/djangoapps/contentstore/views/tests/test_group_configurations.py

```diff
@@ -261,7 +262,7 @@ def test_view_index_ok(self):
 
         response = self.client.get(self._url())
         self.assertEqual(response.status_code, 200)
-        self.assertContains(response, 'First name')
+        self.assertContains(response, 'First name', count=1)
```

Comment:
Without the bug fix, this fails with > 1 occurrence of the split test configuration name.